thavalon-server/src/database/games/db_game.rs

```diff
@@ -73,6 +93,7 @@ impl DatabaseGame {
             _id,
             players: HashSet::with_capacity(10),
             display_names: HashSet::with_capacity(10),
+            players_to_display_names: HashMap::with_capacity(10),
```

Comment:
10 should be a const of MAX_NUM_PLAYERS, since it's used for all 3 of these hashset/hashmap capacities